### ISSUE TITLE
Estimated time

### ISSUE THREAD

**Issue by serhio7** [deepmipt/DeepPavlov#845]
Hello!
How can I find out estimated time left during training process in NER network

**Comment by mu-arkhipov** [comment]
Hello!
We use learning rate scheduling and early stopping strategy based on validation patience. Due to this fact exact training time cannot be calculated. You can approximately estimate maximal training time by multiplying time spent on one epoch by maximal number of epochs (section `["train"]["epochs"]` of the config).
